### PR TITLE
Add jOOQ add on

### PR DESCRIPTION
**Pull request by mosabua**
Related to https://github.com/jOOQ/jOOQ/issues/11485 and the release of jOOQ 3.19.

What do you think @lukaseder ... we can add a bit more details or a links to specific pages as you see useful.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -719,6 +719,21 @@
       url: https://trino.io/docs/current/admin/jmx.html
     - urltext: JMX connector
       url: https://trino.io/docs/current/connector/jmx.html
+- name: jOOQ
+  anchor: jooq
+  category: add-on
+  logo: /assets/images/logos/jooq.png
+  logosmall: /assets/images/logos/jooq-small.png
+  description: |
+    Java Object Oriented Querying (jOOQ) generates Java code from your database
```
Comment (lukaseder) [inline _data/tools.yml:728]: jOOQ stands for jOOQ Object Oriented Querying

**Comment by lukaseder** [comment]
> What do you think @lukaseder ... we can add a bit more details or a links to specific pages as you see useful.

I don't know how or where this renders, but I guess it's good enough from the text itself? jOOQ doesn't do too much trino-specific yet, which is worth mentioning here, I think.

File: _data/tools.yml (hunk 1 above)
Comment (mosabua) [reply _data/tools.yml:728]: Haha .. awesome .. another recursive acronym!

**Comment by mosabua** [comment]
I updated the text a bit more. You can see a preview of the content at

**Review by dain — APPROVED** [review]

**Comment by mosabua** [comment]
Thanks for the approval @dain

**Pull request merged**